sfcasts/ep2-security/production-docs.md

```diff
@@ -143,5 +143,9 @@ you *could* always create an event subscriber on the `kernel.request` event -
 now called the `RequestEvent` in Symfony 4.3 - and, if the URL is `/api` *and*
 the request format is HTML, return a 404.
 
+***TIP
+You can see an example here : [https://bit.ly/2YmR6Uh](https://bit.ly/2YmR6Uh)
```

Comment:
Can we put `:` close to `here`, i.e. `here: [https://bit.ly/2YmR6Uh](https://bit.ly/2YmR6Uh)`? :p

Reply:
@bocharsky-bw grrr, english syntax 😠 is it good for you now ?

Reply:
Yes, thank you!